neurons/validator.py

```diff
@@ -3876,7 +3890,7 @@ def _training_pool_digest(self, uid: int, window: int) -> tuple[str, int]:
             batch_size=self.hparams.batch_size,
             target_batch_size=self.hparams.target_batch_size,
             rank=0,
-            world_size=1,
+            world_size=self.world_size,
```

Comment:
Wouldn't we need 1 here still to sample all the data? I mean it would be the same if tp_degree=world_size but for other cases we won't sample all the data right?

Reply:
Thinking of it a bit more. 1 might have been a bug actually